src/common-ui/containers/IndexDropdown.js

```diff
@@ -66,6 +67,30 @@ class IndexDropdownContainer extends Component {
         return this.props.url != null
     }
 
+    /**
+     *
+     */
+    async storeTrackEvent(isAdded) {
+        let { source } = this.props
+        const { hover } = this.props
+
+        // Make first letter capital
+        source = source.charAt(0).toUpperCase() + source.substr(1)
+
+        // Only for add and remove from the popup or overview, we have already covered filter in overview
+        if (this.allowIndexUpdate) {
+            internalAnalytics.processEvent({
+                type: hover
```

Comment:
I would try to avoid the nested ternary condition operators `... ? ... : ...` as it ends up hard-to-read.